references/classification/utils.py

```diff
@@ -72,3 +72,21 @@ def plot_recorder(lr_recorder, loss_recorder, beta: float = 0.95, **kwargs) -> N
     plt.ylim(vals[min_idx] - 0.1 * delta, max_val + 0.2 * delta)
     plt.grid(True, linestyle="--", axis="x")
     plt.show(**kwargs)
+
+
+class EarlyStopper:
+    def __init__(self, patience: int = 5, min_delta: int = 0):
```

Comment:
min_delta should be a float we can default to 0.01 but this value should be set in the training scripts as default while creating the EarlyStopper object :)

Reply:
Oh by default do you mean to add a parser argument ?

Reply:
Mhh yes lets add a parser argument which defaults also to 0.01

Reply:
I have added that